src/app/Core/ExportServices/generateExportLabels.service.ts

```diff
@@ -2,7 +2,7 @@ import { useTranslation } from "react-i18next";
 import { getCurrentDate } from "../getCurrentDate.service";
 import { MainFilterLabelInterface } from "../../Shared/Model/Export.model";
 
-export interface ExportLabels {
+export interface ExportLabelsInterface {

```

Comment:
better was ExportLabels, we had that a long time ago, adding Interface does not add any meaning to the name. You name interfaces like you would name the actual object it represents, so ExportLabels

Reply:
Done. Commit: "Remove "Interface"-Label"